File: app/src/main/java/com/jqorz/planewar/frame/GameAbout.java
package com.jqorz.planewar.frame;

import android.app.Activity;
import android.os.Bundle;
import android.view.View;

import com.jqorz.planewar.R;

public class GameAbout extends Activity {


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.game_about);
    }

    public void mClick(View v) {
        this.finish();
    }


}

File: app/src/main/java/com/jqorz/planewar/frame/GameAbout.kt
package com.jqorz.planewar.frame

import android.os.Bundle
import android.view.View
import androidx.fragment.app.FragmentActivity
import com.jqorz.planewar.R

class GameAbout : FragmentActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.game_about)
    }

    fun mClick(v: View) {
        finish()
    }
}